microcosm_postgres/store.py

```diff
@@ -41,6 +41,9 @@ def __init__(self, graph, model_class, auto_filter_fields=()):
             auto_filter_field.name: auto_filter_field
             for auto_filter_field in auto_filter_fields
         }
+        self.assign_model_class_store()
```

Comment:
maybe check if `model_class` is null / make `assign_model_class_store` an attribute flag instead? seems a bit weird to override a function in the base class to do nothing (assuming there's no real customization here except skipping)

Reply:
actually there's no point supporting null `model_class` generally. continuing the discussion elsewhere, but seems like a flag either as a parameter or a property might be a bit nicer (assuming the main "real" use case for this will be multiple stores using the same model class and overriding the link)